gen_protos.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # simple sanity checking for repo
-if [ ! -d "../Meshtastic-protobufs" ]; then
+if [ ! -d "../protobufs" ]; then
   echo "Please check out the https://github.com/meshtastic/Meshtastic-protobufs the parent directory."
   exit
```

Comment:
The path is still pointing to the old URL, even if it is just a hint...

Reply:
That should be correct, as that's the folder that the git submodule clones to, but please let me know if I'm missing something

Reply:
I was targetting line 5 ... the echo :-)

Reply:
Lol, my bad, mobile app doesn't show the exact line 🤦

Reply:
Sorry to be so nitpicky, but a new developer onboarding will stumble across this. When i started with the firmware, i was searching 2 days for the nRF includes till i found out i had to check out submodules...

Reply:
Agreed, thanks